chat-plugins/datasearch.js

```diff
@@ -162,12 +162,12 @@ exports.commands = {
 	},
 	movesearchhelp: ["/movesearch [parameter], [parameter], [parameter], ... - Searches for moves that fulfill the selected criteria.",
 		"Search categories are: type, category, contest condition, flag, status inflicted, type boosted, and numeric range for base power, pp, and accuracy.",
-		"Types must be followed by ' type', e.g., 'dragon type'.",
-		"Stat boosts must be preceded with 'boosts ', e.g., 'boosts attack' searches for moves that boost the attack stat.",
+		"Types must be followed by 'type', e.g., 'dragon type'.",
+		"Stat boosts must be preceded with 'boosts', e.g., 'boosts attack' searches for moves that boost the attack stat.",
```

Comment:
The spaces are intentional, it clarifies that it should be "dragon type" and not "dragontype".